### DISCUSSION TITLE
Machinaris 8.2 issues with Ecostake fork - incorrect TCP port number

### DISCUSSION THREAD

**Discussion by LeroyINC** [guydavis/machinaris#740] [General]
Ran into another issue where it looks like the full\node farmer is actually listening on port 38441 and not 38448
so I had to update the docker-compose file for that port.

then I also had to modify the config.yml file on each harvester for the correct port as well... just an FYI

but I believe I have it working now...

**Comment by guydavis** [comment]
Good catch. Sorry for the trouble.  Fixed in next release.
